### PR TITLE
BIOWDL-420: Add QC steps to pipeline.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md.
- [x] Documentation was updated (if required).
- [x] Workflow `parameter_meta` was added/updated (if required).

### CODE REVIEW

**Comment by JasperBoom** [comment]
The tasks I have created for this PR will probably fit better separately in our tasks repository (so I will have to take care of that first, before any merging), but this made testing (and possibly reviewing) easier.

File: sample.wdl [hunk 1]
```diff
@@ -65,38 +96,129 @@ workflow SampleWorkflow {
...
         outputTranscriptCleanLog: {description: "Log file(s) of TranscriptClean run."}
         outputTranscriptCleanSAM: {description: "SAM file(s) containing corrected aligned reads."}
         outputTranscriptCleanTElog: {description: "TE log file(s) of TranscriptClean run."}
+        outputTranscriptCleanBAM: {description: "TranscriptClean BAM file(s) converted from SAM file(s)."}
+        outputTranscriptCleanSortedBAM: {description: "TranscriptClean BAM file(s) sorted on position."}
+        outputTranscriptCleanSortedBai: {description: "Index of sorted TranscriptClean BAM file(s)."}
+    }
+}
+
+task SortAndIndex {
```
Comment (DavyCats) [inline sample.wdl]: Didn't this task already exist in samtools.wdl? Either way it should be in there rather than here.

File: sample.wdl [hunk 2]
```diff
@@ -65,38 +96,129 @@ workflow SampleWorkflow {
...
+
+        Int cores = 1
+        String memory = "2G"
+        String dockerImage = "quay.io/biocontainers/samtools:1.10--h9402c20_2"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputPrefix})"
+        samtools view \
```
Comment (DavyCats) [inline sample.wdl]: Is this `view` command needed? `sort` can take SAM format as input

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: sample.wdl (hunk 1 above)
Comment (DavyCats) [reply sample.wdl]: Ah didn't notice your comment, nvm :p

Comment (JasperBoom) [reply sample.wdl]: There is for indexing indeed, but there was some bash stuff around it which didn't give me the output I was looking for, so I create this, so I was able to continue testing. I will look into it now.

Comment (JasperBoom) [reply sample.wdl]: Sorting seems to only be for name sorting.

File: sample.wdl (hunk 2 above)
Comment (JasperBoom) [reply sample.wdl]: I did not know that, I will test it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,11 @@ that users understand how the changes affect the new version.
 
 version develop
 ---------------------------
```
Comment (DavyCats) [inline CHANGELOG.md:12]:
```suggestion
---------------------------
+ Renamed workflow from `pipeline` to `TalonWDL`.
```

**Review by DavyCats — APPROVED** [review]

**Pull request merged**